package.json

```diff
@@ -1,37 +1,37 @@
 {
-  "name"        : "request-debug",
-  "author"      : "James Nylen <jnylen@gmail.com>",
-  "description" : "Library to assist with debugging HTTP(s) requests made by the request module.",
-  "version"     : "0.2.0",
-  "repository"  : {
-    "type" : "git",
-    "url"  : "https://github.com/request/request-debug"
+  "name": "request-debug",
+  "author": "James Nylen <jnylen@gmail.com>",
+  "description": "Library to assist with debugging HTTP(s) requests made by the request module.",
+  "version": "0.2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/request/request-debug"
   },
-  "bugs" : {
-    "url" : "https://github.com/request/request-debug/issues"
+  "bugs": {
+    "url": "https://github.com/request/request-debug/issues"
   },
-  "license"  : "MIT",
-  "keywords" : [
+  "license": "MIT",
+  "keywords": [
     "request",
     "debug",
     "http",
     "https",
     "headers"
   ],
-  "scripts" : {
-    "test" : "node node_modules/.bin/mocha"
+  "engines": {
+    "node": ">=4"
   },
-  "main"            : "index.js",
-  "devDependencies" : {
-    "detect-engine" : "~1.0.2",
-    "express"       : "~4.8.6",
-    "mocha"         : "~1.21.4",
-    "passport"      : "~0.2.0",
-    "passport-http" : "~0.2.2",
-    "request"       : "~2.50.0",
-    "should"        : "~4.0.4"
+  "scripts": {
+    "test": "mocha"
   },
-  "dependencies" : {
-    "stringify-clone": "^1.0.0"
+  "main": "index.js",
+  "dependencies": {
+    "clone": "^1.0.2",
+    "request": "^2.75.0"
```

Comment:
I think it would be good to remove the npm dependency and allow the user to pass in a `request` instance.  You should be able to do that and keep the same concept of wrapping rather than modifying `request`.  I probably won't have time to do a detailed review but that seems solid to me.